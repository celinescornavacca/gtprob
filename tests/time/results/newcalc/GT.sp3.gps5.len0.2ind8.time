0.31user 0.11system 0:00.62elapsed 69%CPU (0avgtext+0avgdata 43068maxresident)k
133inputs+0outputs (0major+16656minor)pagefaults 0swaps
