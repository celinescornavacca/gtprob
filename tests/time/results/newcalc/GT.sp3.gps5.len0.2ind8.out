-24.3855371348
